docs/pages/kubernetes-access/manage-access/rbac.mdx

```diff
@@ -0,0 +1,452 @@
+    kind: kube_cluster
+    metadata:
+      labels:
+        platform: minikube
+        region: local
+      name: minikube
+    spec:
+      aws: {}
+      azure: {}
+      gcp: {}
+    version: v3
+  host_id: 00000000-0000-0000-0000-000000000000
+  hostname: remote.kube.proxy.teleport.cluster.local
+  rotation:
+    current_id: ""
+    last_rotated: "0001-01-01T00:00:00Z"
+    schedule:
+      standby: "0001-01-01T00:00:00Z"
+      update_clients: "0001-01-01T00:00:00Z"
+      update_servers: "0001-01-01T00:00:00Z"
+    started: "0001-01-01T00:00:00Z"
+  version: (=teleport.version=)
+version: v3
+```
+
+## Step 2/3. Define a Teleport role
+
+The Teleport Kubernetes Service determines how to proxy a Teleport user's
+requests to a Kubernetes API server by looking up the user's roles. Based on
+this information, the Kubernetes Service accepts or denies the request. 
+
+For valid requests, the Kubernetes Service rewrites the request headers to
+impersonate the Teleport user's desired Kubernetes user and groups, and forwards
+the request to the API server.
+
+In this section, we will define a Teleport role that:
+
+- Authenticates the user to a Kubernetes cluster as a member of the `developers`
+  group. In the previous section, we authorized members of this group to view
+  pods in all namespaces. 
+- Enables the user to access `webapp` pods in the `production` namespace and all
+  pods in the `development` namespace.
+- Denies the user access to all other pods.
+
+### Define a role
+
+Create a file called `kube-access.yaml` with the following content:
+
+```yaml
+kind: role
+metadata:
+  name: kube-access
+version: v6
+spec:
+  allow:
+    kubernetes_labels:
+      'region': '*'
+      'platform': 'minikube'
+    kubernetes_resources:
+      - kind: pod
+        namespace: "production"
+        name: "^webapp-[a-z0-9-]+$"
+      - kind: pod
+        namespace: "development"
+        name: "*"
+    kubernetes_groups:
+    - developers
+    kubernetes_users:
+    - minikube
+  deny: {}
+```
+
+In this role, we have defined the following `allow` rules:
+
+- `kubernetes_labels`: Allows access to Kubernetes clusters in all regions, but
+  only with the `platform:minikube` label.
+- `kubernetes_resources`: Allows access to pods in the `webapp` deployment in
+  the `production` namespace and all pods in the `development` namespace. Note
+  the use of a regular expression (beginning `^` and ending in `$`) to match pod
+  names that Kubernetes generates automatically.
+- `kubernetes_groups`: Authenticates the user to your Kubernetes cluster as a
+  member of the Kubernetes group `developers`, which we associated with the
+  `pod-viewer` Kubernetes `Role` earlier in this guide.
+- `kubernetes_users`: Authenticates the user to your Kubernetes cluster as the
+  default `minikube` user.
+  
+### Create the role
+
+Once you have finished configuring the `kube-access` role, create it using the
+following command:
+
+```code
+$ tctl create kube-access.yaml
```

Comment:
Since this is the first time we create the role in this page, I didn't include the `-f`/`--force` flag. Maybe we should leave it out in case the user has accidentally defined a `kube-access` role resource already?

Reply:
Fair enough. I only suggested it because I thought we were using `-f` as a convention, but your argument against it in this case is valid.